index.d.ts

```diff
@@ -0,0 +1,745 @@
+/**
+ * An interface that describes what structure is expected from Snapdragon's Node.
+ */
+declare interface NodeLike<T> {
+    /**
+     * Property that is always present and true in Node.
+     */
+    readonly isNode: boolean;
+
+    /**
+     * [Optional] Value property.
+     */
+    //TODO: Which one is default or legacy?
+    value?: string;
+
```

Comment:
Default is `value`. I thought I replaced all occurrences of `val`, but I just noticed at least one place in a code comment where `val` is used.